### ISSUE TITLE
Retain original XML identifiers

### ISSUE THREAD

**Issue by robin-garnham** [mwilliamson/mammoth.js#355]
Hi there

Great library, thanks to everyone who has contributed.

I'm wondering if there is a way to preserve the original XML ID's onto the generated HTML? 
I can see in the original .docx file that most of the elements have ID's on them, how do I get those onto the generated HTML?

**Comment by mwilliamson** [comment]
No, this isn't something that Mammoth supports, and is probably unlikely to. Would you mind explaining what your use case is?

**Comment by robin-garnham** [comment]
I want to be able to retain the XML identifier so I can confidently write data back into the original Word Doc in the correct location.

I'm happy to fork the library if need be, any tips on where I'd need to change?

**Comment by robin-garnham** [comment]
Yes, I did manage to. I just changed the code and left it as a patch, which worked against 1.6.0 (newer versions might need the patch to be updated).

The code is a bit messy, but it works for my use case :)

[mammoth+1.6.0.patch](https://github.com/mwilliamson/mammoth.js/files/13642075/mammoth%2B1.6.0.patch)

**Comment by christianvoigt** [comment]
thank you for the patch!
